CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ The format is based on [Keep a Changelog][] and this project adheres to
 ## [Unreleased][]
 
 ### Added
+* Add partial support for Braze catalogs operations :
+    * create_catalogs
+    * list_catalogs
+    * delete_catalog
+    * create_catalog_items
+    * update_catalog_items
+    * delete_catalog_items
```

Comment:
This can be simplified like so:

```
* Partial support for Braze catalogs ([#60][])
```

And then down towards the bottom please add:

```
[#61]: https://github.com/jonallured/braze_ruby/pull/61
```